Controllers/deleteController.js

```diff
@@ -0,0 +1,8 @@
+const { deleteGivenComment } = require("../Models/deleteModel");
+
+exports.deleteCommentById = (req, res) => {
+  const { comment_id } = req.params;
+  deleteGivenComment(comment_id).then((deleted) => {
+    res.status(200).send({ deleted });
```

Comment:
See ticket, empty body to be sent in response. Would also recomend a catch block for 500 errors.